src/middlewared/middlewared/etc_files/scst.conf.mako

```diff
@@ -45,6 +45,14 @@
 
     # FIXME: SSD is not being reflected in the initiator, please look into it
     # FIXME: Authorized networks for initiators has not been implemented yet, please look for alternatives in SCST
+
+    target_hosts = defaultdict(list)
+    for row in middleware.call_sync('datastore.query', 'services.iscsihosttarget'):
```

Comment:
Do we need direct datastore query here?

I try to avoid datastore.query as its a "private" API and use the iscsi service interfaces